crates/burn-core/src/nn/leaky_relu.rs

```diff
@@ -0,0 +1,41 @@
+use crate as burn;
+use crate::config::Config;
+use crate::module::Module;
+use crate::tensor::backend::Backend;
+use crate::tensor::Data;
+use crate::tensor::Tensor;
+/// Parametric Relu layer.
+#[derive(Module, Debug)]
+pub struct LeakyRelu<B: Backend> {
+    /// The weight used in Leaky ReLu
+    pub negative_slope: Tensor<B, 1>,
+}
+/// Configuration to create a [Leaky Relu](LeakyRelu) layer.
+#[derive(Config, Debug)]
+pub struct LeakyReluConfig {
+    /// The learnable weight alpha. Default is 0.01
```

Comment:
It isn't learnable right?

Reply:
Yeah non learnable. That was left by mistake